src/service/quirks.rs

```diff
@@ -268,6 +268,7 @@ fn load_quirks() -> HashMap<String, Quirk> {
         Quirk::lan_api_capable_light("H6076", FLOOR_LAMP),
         Quirk::lan_api_capable_light("H6078", FLOOR_LAMP),
         Quirk::lan_api_capable_light("H6087", WALL_SCONCE),
+        Quirk::lan_api_capable_light("6097", TV_BACK),
```

Comment:
Isn't the model supposed to start with "H"? So I think Govee TV Backlight 3 Lite's model should be `H6097`.